Automattic/wp-calypso
# Add thank you screen for akismet checkout

## Proposed Changes

This PR adds a thank you screen when you get finished buying an Akismet product at the route `/checkout/akismet/thank-you/:productSlug`

Worked with @ballio2000 on this design

Note: We plan to show the API key here in the future

## Testing Instructions

1. Checkout this branch
2. Sandbox `public-api.wordpress.com`
3. In your 0-sandbox.php file, add the following line
```php
define( 'USE_STORE_SANDBOX', true );
```
4. Start your local environment via `yarn start`
5. Head to http://calypso.localhost:3000/checkout/akismet/ak_plus_yearly_3
6. Make sure the styles look good on desktop and mobile
![image](https://user-images.githubusercontent.com/65001528/226464081-1c367abd-c848-461b-81f6-7e1b09ccdf39.png)
![image](https://user-images.githubusercontent.com/65001528/226464236-612c11cf-d9a1-429a-8684-52d41ef41867.png)

## Pre-merge Checklist
